src/components/ui/color-mode.tsx

```diff
@@ -11,7 +10,12 @@ export interface ColorModeProviderProps extends ThemeProviderProps {}
 
 export function ColorModeProvider(props: ColorModeProviderProps) {
 	return (
-		<ThemeProvider attribute="class" disableTransitionOnChange {...props} />
+		<ThemeProvider
+			attribute="class"
+			defaultTheme="light"
+			disableTransitionOnChange
+			{...props}
+		/>
```

Comment:
We can just force the theme to be "light" instead of having a default. I think the prop is `forcedTheme="light"`. This will just remove the need to even support the `useColorMode` hook. It is very unlikely that we will even make a dark theme.